.env_sample

```diff
@@ -0,0 +1 @@
+export SENDGRID_API_KEY=''
```

Comment:
@jericho, can you please help me out - you know csharp better than me. WAY better. What's the most appropriate way to create an environment file for C# on windows?

Reply:
@gabrielgene Can you set this file up like this one? https://github.com/sendgrid/sendgrid-csharp/pull/599/files#diff-04c6e90faac2675aa89e2176d2eec7d8R90

Reply:
@mbernier `env` files are a linux and/or docker concept aren't they? Anyway, I'm not familiar with them. The way I personally deal with environment variables is to have a `ps1` powershell file that I execute once to set my variable (this file contains lines such as: `Environment.SetEnvironmentVariable("the_name_of_my_variable", "the_value");`) and then I can retrieve the content of these variables in my C# code using `var value = Environment.GetEnvironmentVariable("the_name_of_my_variable");`

Reply:
NEAT, thank you @Jericho